InkyPhat/app/src/main/java/com/novoda/inkyphat/ShapeDrawer.java

```diff
@@ -0,0 +1,422 @@
+package com.novoda.inkyphat;
+
+import static java.lang.Math.cos;
+import static java.lang.Math.sin;
+
+public class ShapeDrawer {
```

Comment:
yeah so people could use `Inkphat.drawImage(Bitmap` you mean?

Having this class is extra you're right. Could potentially be removed (then in the demo's show shapes being drawn using Canvas).

I guess this class comes in handy if we implemented the "partial screen update" feature... leaning towards dropping it.